server/src/domain/auth/auth.service.spec.ts

```diff
@@ -543,7 +543,7 @@ describe('AuthService', () => {
         loginResponseStub.user1oauth,
       );
 
-      expect(userMock.create).toHaveBeenCalledWith(userDto.userWithDefaultStorageQuota);
+      expect(userMock.create).toHaveBeenCalledWith(userDto.userWithUnsetQuotaClaim);
```

Comment:
It might be better to remove these user objects and just assert it matches an object containing the right quota value.

Reply:
Yeah I can make that change